terraform/web.tf

```diff
@@ -135,8 +135,11 @@ locals {
   website_config_origin_path  = "config"
   website_config_object_key   = "${local.website_config_origin_path}/deploy-config.js"
   website_config_object_contents = templatefile("${path.module}/tpl/deploy-config.js", {
-    feature_flags     = jsonencode(var.website_feature_flags),
-    web_config_params = jsonencode(var.website_config_params),
+    feature_flags = jsonencode(var.website_feature_flags),
+    web_config_params = jsonencode(merge(
+      { auth_provider = "passage", passage_app_id = var.passage_app_id },
+      var.website_config_params,
+    )),
```

Comment:
Instead of needing to provide `passage_app_id` twice (once as an input variable and again in `website_config_params`, this change uses `var.passage_app_id` directly. The net effect is that it will use `var.passage_app_id` by default, but still allows both `auth_provider` and `passage_app_id` key/value pairs to be overridden if conflicting keys are provided in `var.website_config_params`.